drivers/config.go

```diff
@@ -100,6 +103,8 @@ func (c Config) Int(key string) (int, bool) {
 		integer = t
 	case float64:
 		integer = int(t)
+	case string:
+		integer, _ = strconv.Atoi(t)
```

Comment:
Ignoring the error here is worse than above because we have a way to signal failure in this method (the bool return). If the parsing fails, then your port will inexplicably be 0 without any failure notifications.

Reply:
Fixed in 6722d57ab31fffadaf70e219e0deb7d4ac4406c8